build.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 name: "jellyfin-plugin-ldapauth"
 guid: "958aad66-3784-4d2a-b89a-a7b6fab6e25c"
-version: "6" # Please increment with each pull request
-jellyfin_version: "10.4.0" # The earliest binary-compatable version
+version: "7" # Please increment with each pull request
+jellyfin_version: "10.5.0" # The earliest binary-compatable version
```

Comment:
```suggestion
version: "7"
jellyfin_version: "10.5.0"
```

Reply:
We should just write this in the README or something, because there are only two scenarios.

1.) Someone knows they have to update these and the comments are useless.
2.) They aren't aware and thus have no reason to check this file, which means the comments aren't discoverable.